netman/adapters/switches/brocade.py

```diff
@@ -463,6 +472,8 @@ def add_interface_vlan_data(target_vlan, int_vlan_data):
             vrrp_group = None
         elif regex.match("^ ip helper-address ([^\s]*)", line):
             target_vlan.dhcp_relay_servers.append(IPAddress(regex[0]))
+        elif regex.match("^\s*no ip redirect", line):
```

Comment:
I saw a commit in fake switch addressing a space problem about this command.

The switch has 1 or 2 space?